b3lb/rest/endpoints/b3lb/endpoints.py

```diff
@@ -221,25 +283,24 @@ async def create(request, endpoint, params, node, secret):
 
     params["meta_endCallbackUrl"] = "https://{}-{}.{}/b3lb/b/meeting/end?nonce={}".format(secret.tenant.slug.lower(), str(secret.sub_id).zfill(3), settings.B3LB_API_BACKEND_DOMAIN, meeting.nonce)
```

Comment:
We should **not** require wildcard domains on the backend. There is no requirement to make those callbacks distinguishable at all since we are using a collision resistant cryptographic hashing function.